Packs/opswat-metadefender/Integrations/OPSWATMetadefenderV2/OPSWATMetadefenderV2.py

```diff
@@ -36,7 +35,12 @@ def http_req(method='GET', url_suffix='', file_name=None, parse_json=True):
     if USE_CLOUD:
         headers['apikey'] = API_KEY
     if file_name:
-        headers['filename'] = file_name
+        if type(file_name) == str:
+            demisto.info("in string")
+            headers['filename'] = file_name.encode('utf-8')  # type: ignore
+        else:
```

Comment:
double check that we always get string here, and lets remove this if case. Better not to have more things that future devs will need to maintain if we dont really think theres a need for it here